examples/templates/api/change-algorithm-suite.md

```diff
@@ -0,0 +1,133 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Changing the algorithm suite example
+
+Implementations of this example MUST follow the rules defined in
+[Example Templates](../../../examples.md#example-templates).
+
+## Implementations
+
+- [Python (DEV)](https://github.com/aws/aws-encryption-sdk-python/blob/keyring/examples/src/onestep_unsigned.py)
+- [Java (DEV)](https://github.com/aws/aws-encryption-sdk-java/blob/keyring/src/examples/java/com/amazonaws/crypto/examples/OneStepUnsigned.java)
+
+## Definitions
+
+### Conventions used in this document
+
+The key words
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Header
+
+```python
+# This example shows how to specify an algorithm suite
+# when using the one-step encrypt and decrypt APIs.
+#
+# In this example, we use an AWS KMS customer master key (CMK),
+# but you can use other key management options with the AWS Encryption SDK.
+# For examples that demonstrate how to use other key management configurations,
+# see the ``keyring`` and ``master_key_provider`` directories.
+#
+# The default algorithm suite includes a message-level signature
```

Comment:
If the "default algorithm suite" ever changes, how are we going to find all these places to update them?